docs/development/GettingStarted.rst

```diff
@@ -5,6 +5,7 @@ Setup the Project
 Installation
 ~~~~~~~~~~~~
 - Clone the project.
+- From project root, execute `npm i`.
```

Comment:
IMHO, i recommend to use `npm i` once and followed by `npm ci`

Reference: https://www.geeksforgeeks.org/difference-between-npm-i-and-npm-ci-in-node-js/

Reply:
`npm ci` should be used only for production/build stages

Reply:
For local development it's quite usefull as well. Sometimes with `npm i` what happens is if a package is updated to a new version both the old and new version files will be available in node_modules folder where with ci, it might be a bit slow but ensures the old node_modules are updated with the new one. 

Might be wrong about the information but do let me know if otherwise.

Reply:
for running locally, in my opinion, `npm i` works. installing dependencies isnt a regular use case for local development. but yes, we can mention this in the doc nevertheless. Added in the doc in commit [e85d6b8](https://github.com/CraftsmenLtd/BloodConnect/pull/44/commits/e85d6b8d99493f4c9828109e33fec89949935f88)

for ci environment, we should, of course, use `npm ci` as pointed out above.